src/evict/evict_lru.c

```diff
@@ -2300,6 +2300,10 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
             if (ret == WT_ROLLBACK) {
                 --cache->evict_aggressive_score;
                 WT_STAT_CONN_INCR(session, txn_fail_cache);
+                if (timer)
+                    WT_ERR(__wt_msg(session, "%s",
```

Comment:
WT_IGNORE_RET would be better here as we'd persist the WT_ROLLBACK value which would then be returned as soon as we exit the loop.

Reply:
Changed